src/repository/index.ts

```diff
@@ -97,7 +105,7 @@ export default class Repository extends EventEmitter implements EventEmitter {
 
   private eventSource: EventSource | undefined;
 
-  private initialEventSourceConnected: boolean = false;
```

Comment:
no need to check if first connection is different from another since all streaming goes with SSEs